src/components/FiltersDrawer/FiltersDrawer.tsx

```diff
@@ -37,6 +43,11 @@ export type FiltersDrawerProps = {
    * Callback called when the apply button is called.
    */
   onApply?: () => void;
+  /**
+   * Trigger element that toggles the filters drawer.
+   * Must be able to accept an 'onClick: MouseEventHandler' prop.
+   */
```

Comment:
Maybe add something in the prop comments about how consumers need to pass in either `triggerElement` or `triggerText` but not both.

Reply:
By `Must be able to accept an 'onClick: MouseEventHandler' prop.` do you mean it shouldn't have that prop already? I'm asking because I'm wondering if line 128 would override any `onClick` already being passed in.

Reply:
> Maybe add something in the prop comments about how consumers need to pass in either triggerElement or triggerText but not both (and that if both are passed in, only the triggerElement will be used).

Sounds good, will add comment

> By Must be able to accept an 'onClick: MouseEventHandler' prop. do you mean it shouldn't have that prop already? I'm asking because I'm wondering if line 128 would override any onClick already being passed in.

In line 128, `() => setIsActive(true)` is being passed to the `triggerElement` as `onClick`, so `triggerElement` needs to be able to accept the `onClick` function.  The comment can be changed to `Must be able to accept an 'onClick: () => void' prop.`

Reply:
Damn good question. I made sure to call the original `onClick`, as shown: `console.log('yeehaw')` is original `<button>` `onClick`:

https://user-images.githubusercontent.com/86632227/192010584-5c436fa4-16dd-4086-bc06-8780843bf7bb.mov

Reply:
Awesome, thanks!